cmd/sriov-network-config-daemon/start.go

```diff
@@ -150,17 +152,34 @@ func runStartCmd(cmd *cobra.Command, args []string) {
 		destdir = "/host/tmp"
 	}
 
-	platformType := utils.Baremetal
+	backoff := wait.Backoff{
```

Comment:
We would like to retry if possible, maybe with a shorter timeout  if that works better for you. In managed environments, the apiserver may be undergoing maintenance or other events that make it temporarily  unavailable, and it would be a better experience if we retry instead of panicing right a way.

Reply:
I still dont understand why cant this panic then, since its deployed as daemonset, it would restart.